docs/site/translation.md

```diff
@@ -7,25 +7,29 @@ title: Information Hub for Linguists
 💡🆕
 
 The sequence above marks items that have been recently added. In your browser you can copy this sequence, then use ⌘-F (Mac) or Ctrl-F (Windows) to find all the places it occurs.
+- 💡🆕 Added missing date & time formats:
+  - GyM and GyMEd
+  - Time formats: EBh, Eh, EH when there is an existing EBhm, Ehm, and EHm in the calendar respectively.
+  - Changed HH patterns in available and internval formats to include a reference to hour since seeing an hour number alone is ambigious.
```

Comment:
```suggestion
  - Changed HH patterns in available and internal formats to include a reference to hour since seeing an hour number alone is ambigious.
```